sw/device/tests/aes_test.c

```diff
@@ -4,9 +4,9 @@
 
 #include "sw/device/lib/aes.h"
 
-#include "sw/device/lib/arch/device.h"
-#include "sw/device/lib/common.h"
-#include "sw/device/lib/uart.h"
+#include "sw/device/lib/base/log.h"
```

Comment:
This commit needs to be merged with the meson path update or else they're not a single cohesive unit.

Reply:
Done.